index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Appl Old Version</title>
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/navbar.css">
+  </head>
+  <body>
+    <main id="container">
+      <nav class="nav-bar">
+        <div id="nav-initial"><a href="#"><img id="apple-logo" src="https://upload.wikimedia.org/wikipedia/commons/9/9a/Logo_apple_pnh.png"></div>
+        <div><a href="#">Store</a></div>
+        <div><a href="#">Mac</a></div>
+        <div><a href="#">iPod</a></div>
+        <div><a href="#">iPhone</a></div>
+        <div><a href="#">iPad</a></div>
+        <div><a href="#">iTunes</a></div>
+        <div><a href="#">Support</a></div>
+        <div id="nav-final">
+          <img src="images/magnifier-white.png" alt="" id="search-icon" />
+          <input type="text" name="Search" value="" id="nav-search">
+        </div>
+      </nav>
+      <div class="your-verse">
+        <img src="images/your_verse_title_small.png" alt="" />
+        <img src="images/your_verse_subhead_small.png" alt="" id="subhead" />
+        <div class="film-story">
+
+        </div>
+      </div>
+    </main>
+    <div class="wrapper">
```

Comment:
The four boxes inside the container with the "wrapper" class should be linked. 

(the `<div>` could be contained within anchor elements with a # pound to be linked)